core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/QualificationBenchmark.scala

```diff
@@ -0,0 +1,27 @@
+package org.apache.spark.rapids.tool.benchmarks
+
+import com.nvidia.spark.rapids.tool.qualification.QualificationArgs
+import com.nvidia.spark.rapids.tool.qualification.QualificationMain.mainInternal
+
+object QualificationBenchmark extends BenchmarkBase {
```

Comment:
This object needs a main method, right?
Otherwise, we won't be able to create the benchamrk frm the CLI.

Reply:
The main method is taken from BenchmarkBase which in turn calls the overridden runBenchmarkSuite